bp_common/test/src/perch/start.S

```diff
@@ -54,6 +54,10 @@ _start:
     addi x30, x0, 0
     addi x31, x0, 0
 
+.option push
```

Comment:
Could you explain / comment this?

Reply:
gp is used in the LFS test, so it should be initialized in the start code.

Reply:
I see. And the global pointers variable is a generic RISCV gcc concept?

Reply:
Yeah, apparently it points to the symbols at the start of the .bss section.
https://www.sifive.com/blog/all-aboard-part-3-linker-relaxation-in-riscv-toolchain

Reply:
Interesting!